testscommon/enableEpochsHandlerStub.go

```diff
@@ -0,0 +1,707 @@
+package testscommon
+
+type EnableEpochsHandlerStub struct {
```

Comment:
missing comment.

In order to make this struct smaller, you can implement it as:
```
type EnableEpochsHandlerMock struct {
    BlockGasAndFeesReCheckEnableEpochField uint32
    IsSCDeployFlagEnabledField             bool
// ....
}

// BlockGasAndFeesReCheckEnableEpoch -
func (mock *EnableEpochsHandlerMock) BlockGasAndFeesReCheckEnableEpoch() uint32 {
    return mock.BlockGasAndFeesReCheckEnableEpochField
}

// IsSCDeployFlagEnabled -
func (mock *EnableEpochsHandlerMock) IsSCDeployFlagEnabled() bool {
    return mock.IsSCDeployFlagEnabledField
}
````

in this hay this struct will grow far less. Can be done in another PR

Reply:
refactored